lib/docker.js

```diff
@@ -8,18 +8,19 @@ const path = require('path');
  * @param {string[]} options
  * @return {Object}
  */
-function dockerCommand(options) {
+async function dockerCommand(options) {
   const cmd = 'docker';
-  const ps = spawnSync(cmd, options, { encoding: 'utf-8' });
-  if (ps.error) {
-    if (ps.error.code === 'ENOENT') {
+  try {
+    return await spawn(cmd, options, { encoding: 'utf-8' });
+  } catch (e) {
+    if (
+      e.stderrBuffer &&
+      e.stderrBuffer.toString().includes('command not found')
+    ) {
       throw new Error('docker not found! Please install it.');
```

Comment:
It feels as good opportunity to switch to `ServerlessError`, WDYT?

Is this easily possible? Or maybe, to not overcomplicate this PR, better to handle in next (I see it may require an args shuffling)?

Reply:
I was thinking about it but decided against for now to keep things simpler - I'm already changing arguments a little bit in logs PR and I would like to merge that one first (rebase on top of this change is already a bit time-consuming and I would like to avoid complicating it more)

Reply:
Ok, at some point it'll be good to refactor those errors to `ServerlessError` as we receive a lot programmer marked crashes from this plugin in metrics